### DISCUSSION TITLE
How to install maps in a device without internet?

### DISCUSSION THREAD

**Discussion by forofo** [organicmaps/organicmaps#8380] [Q&A] [answered]
I have one android device without internet, wifi or Bluetooth. Only GPS and USB connection.
I wish to install Organic Maps with some selected maps. 

Can I download the maps from one mirror into one USB flash, and later copy into .../Android/data/app.organicmaps/files/  in the device?
It would work just by doing this? Or should I do any addition thing?

(* Excuse me if the question can be trivial for yours. I cannot find some page in the website with instructions. )

**Comment by Misalf-git** [comment, marked as answer]
Pretty much, yes.
Except that maps are stored in a sub-dir of "files", named reflecting the build date of the maps which is tied to app release (hard-coded).
Like `.../Android/data/app.organicmaps.web/files/240528/*.mwm`

**Reply by forofo** [reply]
good news or it would be a problem? :)

Also, there is a .dat file  inside /Android/data/app.organicmaps.web/files/ icudt73l.dat
Do you know if that file should be copied, or this is not necessary?

thanks for the help!

**Reply by Misalf-git** [reply]
No idea about the .dat file.

**Reply by pastk** [reply]
Copying that dat file is not necessary.

**Reply by forofo** [reply]
thanks!

**Comment by PerennialNovice** [comment]
How can I find out which map release is required?

**Reply by biodranik** [reply]
The bottom left About button shows the map data version/snapshot date, you need the data of the same version, e.g. 241001 (October 1, 2024). Here is the mirror: https://cdn.organicmaps.app/

P.S. Don't forget to restart OM after any manual map file operations.

**Reply by PerennialNovice** [reply]
Thanks, but I never made it past the "download world map" screen (with version 241004 from fdroid )
then I tried the apks here on github and it works with the 241008 version and 241001 maps :)

still can't find the bottom left about button ...

**Reply by PerennialNovice** [reply]
oh... its the "logo" button, the left-most button on the right side of the screen :)

**Reply by pastk** [reply]
yeap usually the app version and data version have several days difference
